api/applicant/controllers/applicant.js

```diff
@@ -26,6 +32,7 @@ module.exports = {
       level
     } = ctx.request.body;
 
+    console.log('new application');
 
     const applicantProject = await strapi.services.project.findOne({ slug: project });
```

Comment:
I tested myself, we need to use `strapi.query('project').findOne({ slug: project })` to find the project from database. `strapi.services.project` looks for functions defined in https://github.com/dev-launchers/strapi/blob/main/api/project/services/project.js, but there's isn't a `findOne` function there.

Reply:
Jiawen pointed out that `strapi.services.project.findOne` works for published projects, so let's stick with what you have, since we won't have applications to unpublished projects.